packages/gamut/src/Form/styles/shared-system-props.tsx

```diff
@@ -89,3 +97,113 @@ export const conditionalStyles = system.variant({
 export const conditionalStyleState = (error: boolean, activated: boolean) => {
   return error ? 'error' : activated ? 'activated' : undefined;
 };
+
+export const radioWrapper = system.css({
+  margin: '0.25rem 0',
+  width: '100%',
+  fontWeight: 'normal',
+});
+
+const consistentLabelStyles = {
+  content: '""',
+  display: 'block',
+  width: 20,
+  height: 20,
+  borderRadius: '100%',
+  mr: 8,
+} as const;
+
+export const radioLabel = system.css({
+  display: 'flex',
+  padding: '1rem 0',
+  alignItems: 'center',
+  cursor: 'pointer',
+  position: 'relative',
+  '&::before': {
+    ...consistentLabelStyles,
+    bg: 'background',
+    boxShadow: `0 0 0 1px ${theme.colors.primary}`,
+    transition: timing.slow,
+  },
+  '&::after': {
+    ...consistentLabelStyles,
+    position: 'absolute',
+    transition: `transform ${timing.slow} ease-in-out`,
+    borderWidth: 5,
+    borderStyle: 'solid',
+    borderColor: 'background',
+    transform: 'scale(0)',
+  },
+});
+
+export const radioInput = system.css({
+  '&:checked + label::after': {
+    bg: 'primary',
+    borderWidth: 4,
+    borderStyle: 'solid',
+    borderColor: 'background',
+    transform: 'scale(1)',
+  },
+  '&:checked + label::before': {
+    boxShadow: `0 0 0 1px ${theme.colors.primary}`,
+  },
+  '&:hover + label::before': {
+    boxShadow: `0 0 0 2px ${theme.colors.primary}`,
+  },
+  '&:focus + label::before': {
+    boxShadow: `0 0 0 2px ${theme.colors.primary}`,
+  },
+});
+
+export const conditionalRadioLabelStyles = system.variant({
+  variants: {
+    error: {
+      textColor: 'danger',
+      '&::before': {
+        boxShadow: `0 0 0 1px currentColor`,
+      },
+    },
+    disabled: {
+      ...formFieldBaseDisabledStyles,
+      '&::before': {
+        boxShadow: `0 0 0 1px currentColor`,
+      },
+    },
+  },
+});
+
+export const conditionalRadioInputStyles = system.variant({
+  variants: {
+    error: {
+      '&:checked + label::after': {
+        bg: 'danger',
+      },
+      '&:checked + label::before': {
+        boxShadow: `0 0 0 1px currentColor`,
+      },
+
+      '&:hover + label::before': {
+        boxShadow: `0 0 0 2px currentColor`,
+      },
+      '&:focus + label::before': {
+        boxShadow: `0 0 0 2px currentColor`,
+      },
+    },
+    disabled: {
+      ...formFieldBaseDisabledStyles,
+      '&:checked + label::after': {
+        bg: 'currentColor',
+      },
+      '&:checked + label::before': {
+        boxShadow: `0 0 0 1px currentColor`,
+      },
+
+      '&:hover + label::before': {
+        boxShadow: `0 0 0 2px currentColor`,
+      },
+      '&:focus + label::before': {
+        boxShadow: `0 0 0 2px currentColor`,
+      },
+    },
+  },
+});
```

Comment:
For shared styles the should only be added when a variant is present you can add them to base (since these don't change the actual color just add the style :))

```suggestion
export const conditionalRadioInputStyles = system.variant({
  base: {
      '&:checked + label::before': {
        boxShadow: `0 0 0 1px currentColor`,
      },
      '&:hover + label::before,
       &:focus + label::before': {
        boxShadow: `0 0 0 2px currentColor`,
      },
      '': {
        boxShadow: `0 0 0 2px currentColor`,
      },
   },
  variants: {
    error: {
      '&:checked + label::after': {
        bg: 'danger',
      },
    },
    disabled: {
      ...formFieldBaseDisabledStyles,
      '&:checked + label::after': {
        bg: 'currentColor',
      },
    },
  },
});
```